bin/linters.sh

```diff
@@ -113,14 +113,8 @@ function run_gometalinter() {
 }
```

Comment:
Do we still need the gometalinter above?

Reply:
No. There are three extra linters not enabled in golangci-lint that are enabled in gometalinter. Yet, gometalinter does not detect any violations for whatever reason. These three are documented in .golangci.yml file. It's a monumental effort to enable `errcheck`, for example.

I removed gometalinter since it's not effective and gives a false sense of confidence.